### PR TITLE
Make this repo to hold only tests

### PR DESCRIPTION
**Pull request by machacekondra**
Need to make working unit tests.

### CODE REVIEW

File: lib/ansible/module_utils/kubevirt.py [hunk 1]
```diff
@@ -273,4 +272,4 @@ def execute_crud(self, kind, definition):
         """ Module execution """
         resource = self.find_supported_resource(kind)
         definition = self.set_defaults(resource, definition)
-        return self.perform_action(resource, definition)
\ No newline at end of file
+        return self.perform_action(resource, definition)
```
Comment (mmazur) [inline lib/ansible/module_utils/kubevirt.py]: :arrow_up:

**Comment by machacekondra** [comment]
@mmazur @pkliczewski This PR removes everything and starts to use it from upstream only. The travis job is setup to run everyday. The only module I've left is `cdi_upload`. @mmazur when it's working please submit it to Ansible repo and write a test and remove it from here.

File: README.md [hunk 2]
```diff
@@ -1,59 +1,6 @@
...
----
-- hosts: localhost
-  roles:
-    - role: ansible-kubevirt-modules
-      install_python_requirements: no
-    - role: hello-underworld
-```
-
-Because the role is referenced, the `hello-underworld` role is able to make use of the kubevirt modules.
+Modules have been moved to Ansible repository. This repo now only hold the integration tests of the modules.
```
Comment (pkliczewski) [inline README.md]: Would be good to have a link here to the modules.

Comment (machacekondra) [reply README.md]: added

**Comment by pkliczewski** [comment]
@mmazur review?

**Pull request merged**